scripts/drop-oracle-tables.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export DYLD_LIBRARY_PATH=/opt/oracle/instantclient/
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+/opt/oracle/instantclient/sqlplus system/oracle@192.168.99.100:49161/xe < "$DIR/drop-tables.sql"
```

Comment:
Minor comment:  I would take username and password from the command line arguments, so that you can drop table as a different Oracle user.  But if the purpose of this script is primarily to drop tables as "system" user, that's fine.